src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -454,6 +470,93 @@ public AgentSettings LoadSettings()
             }
         }
 
+#if OS_WINDOWS
+        private void ConfigureAutoLogonIfNeeded(CommandSettings command)
+        {
+            Trace.Info(nameof(ConfigureAutoLogonIfNeeded));
+
+            bool enableAutoLogon = command.GetEnableAutoLogon();
+            if(!enableAutoLogon)
+            {
+                Trace.Info("AutoLogon will not be enabled as per user's input.");
+                return;
+            }
+
+            try
+            {
+                AssertAdminAccess();
+                
+                var iConfigManager = HostContext.GetService<IInteractiveSessionConfigurationManager>();
```

Comment:
Actually I'm thinking that "AutoLogon" terminology may be better than "InteractiveService" terminology. Service means something very specific (e.g. Windows Service) and I worry it will lead to confusion to start reusing the word "Service".